.github/templates/bazel_ci_workflow.yml.j2

```diff
@@ -69,6 +69,9 @@ name: Bazel Linux CI (!{{ build_environment }})
             -w /var/lib/jenkins/workspace \
             "${DOCKER_IMAGE}" \
             sh -c 'sudo chown -R jenkins . && sudo chown -R jenkins /dev && .jenkins/pytorch/build.sh'
+      - name: Parse ref
+        id: parse-ref
+        run: .github/scripts/parse_ref.py
```

Comment:
discovered by the actionlint, many places the step `parse-ref` is missing, and things like `CIRCLE_BRANCH: ${{ steps.parse-ref.outputs.branch }}` is basically empty in the following steps